app/gui2/src/components/GraphEditor/GraphNode.vue

```diff
@@ -405,14 +418,13 @@ watchEffect(() => {
     class="GraphNode"
     :style="{
       transform,
-      minWidth: isVisualizationVisible ? `${visualizationWidth ?? 200}px` : undefined,
+      minWidth: isVisualizationEnabled ? `${visualizationWidth ?? 200}px` : undefined,
```

Comment:
`min-width` not set when previewing?

Reply:
Yes, The previewing is so temporary, that resizing node gave bad feeling. So I decided that the node will be resized only when visualization is actually enabled.